### DISCUSSION TITLE
admin page still active after commenting out ADMIN_TOKEN in docker-compose

### DISCUSSION THREAD

**Discussion by degrashopper** [dani-garcia/vaultwarden#2164] [Q&A] [answered]
### Subject of the issue
admin page

### Deployment environment

* vaultwarden version: 1.23.1

* Install method: docker

* Clients used: desktop, android

* Reverse proxy and version: traefik v2.5.4

* MySQL/MariaDB or PostgreSQL version: SQLite

* Other relevant details:

### Steps to reproduce
comment out ADMIN_TOKEN under vaultwarden in docker-compose and restart the container.
### Expected behaviour
/admin page should not exist after commenting out ADMIN_TOKEN

### Actual behaviour
/admin page is still visitable

**Comment by BlackDex** [comment, marked as answer]
You probably have saved some settings via the /admin interface before, which creates a `config.json` which overrules the ENV variables including the `ADMIN_TOKEN`.

Either delete that file, or empty that out via the /admin interface.

**Reply by degrashopper** [reply]
@BlackDex Thanks for your answer that helped. After what you told me and looking at the file .env.template it was clear to me. Additional setting was added and deactivating the admin page is working again. 
Via the documentation it was not clear (enough) to me that there are 2 ways of setting up admin settings. And that editing via the admin page will overrule the docker environment parameters once the config.json file exists.
